app/controllers/api/members_controller.rb

```diff
@@ -16,4 +22,13 @@ def create
   def member_params
     params.permit(:name, :email, :country, :postal)
   end
+
+  def member_validation
+    [
+      { name: 'email', data_type: 'email', required: true },
+      { name: 'country', data_type: 'country', required: false },
+      { name: 'postal', data_type: 'postal', required: false },
+      { name: 'name', data_type: 'text', required: true }
+    ]
+  end
```

Comment:
I think executing send_to_ak if successful is exactly the kind of logic that should be in the controller - it's just checking that methods called on other classes return the correct values, and if they do, it's calling other methods (`:send_to_ak`, `:errors`) and packing relevant values to send to the client - all orchestration.

I agree that the member_validation is less relevant, but this is a specification used only here. I don't think it makes sense to create a class that is called in one place and does nothing but call another class with a set of parameters... not sure what the best option is.